packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -932,30 +930,28 @@ class CupertinoDynamicColor extends Color with Diagnosticable {
       return null;
     }
     assert(context != null);
-    return (resolvable is CupertinoDynamicColor)
-      ? resolvable.resolveFrom(context)
-      : resolvable;
+    return (resolvable is CupertinoDynamicColor) ? resolvable.resolveFrom(context) : resolvable;
   }
 
   bool get _isPlatformBrightnessDependent {
-    return color != darkColor
-        || elevatedColor != darkElevatedColor
-        || highContrastColor != darkHighContrastColor
-        || highContrastElevatedColor != darkHighContrastElevatedColor;
+    return color != darkColor ||
+        elevatedColor != darkElevatedColor ||
+        highContrastColor != darkHighContrastColor ||
+        highContrastElevatedColor != darkHighContrastElevatedColor;
   }
 
   bool get _isHighContrastDependent {
-    return color != highContrastColor
-        || darkColor != darkHighContrastColor
-        || elevatedColor != highContrastElevatedColor
-        || darkElevatedColor != darkHighContrastElevatedColor;
+    return color != highContrastColor ||
+        darkColor != darkHighContrastColor ||
+        elevatedColor != highContrastElevatedColor ||
+        darkElevatedColor != darkHighContrastElevatedColor;
   }
 
   bool get _isInterfaceElevationDependent {
-    return color != elevatedColor
-        || darkColor != darkElevatedColor
-        || highContrastColor != highContrastElevatedColor
-        || darkHighContrastColor != darkHighContrastElevatedColor;
+    return color != elevatedColor ||
+        darkColor != darkElevatedColor ||
+        highContrastColor != highContrastElevatedColor ||
+        darkHighContrastColor != darkHighContrastElevatedColor;
```

Comment:
This is an example where manually formatted is much better, the most important part of the expression is the `||` which we put on the left, aligned, so it is obvious that this is a "if any of these conditions are true" expression. putting them on the right means I need to scan to the end of different length lines